src/vm/method.cpp

```diff
@@ -2188,6 +2188,12 @@ PCODE MethodDesc::TryGetMultiCallableAddrOfCode(CORINFO_ACCESS_FLAGS accessFlags
     if (HasStableEntryPoint())
         return GetStableEntryPoint();
 
+    if (IsTieredVtableMethod())
```

Comment:
The fallthrough path doesn't always return null. It may return the temporary entry point and for these methods that precode always points to the prestub. This was the only call to `MayHavePrecode()` and I had elected to assert instead of changing it, I have changed it now.